compiler/rustc_target/src/spec/targets/aarch64_apple_watchos.rs

```diff
@@ -0,0 +1,19 @@
+use crate::spec::base::apple::{opts, Arch};
+use crate::spec::{Target, TargetOptions};
+
+pub fn target() -> Target {
+    let base = opts("watchos", Arch::Arm64);
+    Target {
+        llvm_target: "aarch-apple-watchos".into(),
```

Comment:
This is the wrong target name.

```
$ rustc --print cfg --target aarch64-apple-watchos
error: could not create LLVM TargetMachine for triple: aarch-apple-watcho: No available targets are compatible with triple "aarch-apple-watchos"
```

Reply:
Sorry it's my mistake, fix pr: https://github.com/rust-lang/rust/pull/119176